packages/create-astro/src/index.ts

```diff
@@ -272,21 +275,66 @@ export async function main() {
 	spinner.succeed();
 	console.log(bold(green('✔') + ' Done!'));
 
+	const installResponse = await prompts({
+		type: 'confirm',
+		name: 'install',
+		message: `Would you like us to run "${pkgManager} install?"`,
+		initial: true,
+	});
+
+	if (!installResponse) {
+		process.exit(0);
+	}
+
+	if (installResponse.install) {
+		const installExec = exec(`cd ${cwd} && ${pkgManager} install`);
```

Comment:
I'm worried that the `&&` may not work in Windows. I think there is a way to set the cwd of the command